### PR TITLE
SCRUM_4_Added_mar_ t_ test

### PR DESCRIPTION
**Pull request by Jasonn5**
Added mark to categorize tests between regression, functional, and smoke

### CODE REVIEW

File: tests/presentations/test_GetPresentations.py
Comment (AlisonGuzman2006) [inline tests/presentations/test_GetPresentations.py:1]: The obly thing added to the code are the marks, therefore this code is clean and is accepted

**Review by AlisonGuzman2006 — APPROVED** [review]
The changes have been reviewed and accepted

**Review by DiegoClaros17 — APPROVED** [review]
Code marks added works correctly. No objections for me.

**Review by Diego-Claros17 — APPROVED** [review]
Code marks works correctly, no objections for me. Good job.

**Pull request merged**